src/index.css

```
@@ -0,0 +1,134 @@
+@import 'tailwindcss';
+/* @import 'react-day-picker/style.css'; */
+@import 'tw-animate-css';
+@import 'shadcn/tailwind.css';
+@import '@fontsource-variable/geist';
+@import 'tw-animate-css';
```

Stylelint (17.13.0)

[error] Duplicate `@import` rule tw-animate-css (no-duplicate-at-import-rules)

(no-duplicate-at-import-rules)